hodor

```diff
@@ -28,6 +28,12 @@ class Hodor
 
     process_project_config
 
+    if @hostmanager == "boot2docker"
+      @ssh_file = "~/.ssh/id_boot2docker"
+    elsif @hostmanager == "docker-machine"
+      @ssh_file = "~/.docker/machine/machines/#{@host}/id_rsa"
+    end
```

Comment:
You probably forgot top remove this block, since we define @ssh_file in other parts now.